### DISCUSSION TITLE
Successfully use PgBouncer?

### DISCUSSION THREAD

**Discussion by kidoman** [go-pg/pg#1966] [Q&A]
Hey fellow go-pg fans,

Is it a bad idea to use go-pg with PGB? I am reading mixed reports:

- people trying to figure out binary_parameters
- not able to turn off prepared statements
- having random timeout issues

Are there folks here who are successfully using this combo? Would love to hear from you.

**Comment by vmihailenco** [comment]
go-pg/Bun are fully compatible with PgBouncer, because they don't use prepared statements. I am using PgBouncer without any issues.

**Reply by kidoman** [reply]
Thanks. Someone mentioned that we cannot use the transaction pool mode.

What mode would you recommend instead? Session or pool?

**Reply by vmihailenco** [reply]
I recommend to use `pool_mode = transaction`. go-pg works with it just fine.
